CONTRIBUTING.md

```diff
@@ -151,37 +155,50 @@ We parse the titles of all the merged pull requests to generate changelogs. We a
 To pass the *`check-PR-title`* check, the pull request should have the title that follows this schema:
```

Comment:
Let's say that PR titles have to:
- follow Conventional Commits (leave a link to the specification and don't reiterate it here)
- if the PR has a single commit, the PR title must match the commit message